### PR TITLE
Add german translation

### PR DESCRIPTION
**Pull request by hubidu**
- Added a translation for german commands

### CODE REVIEW

File: translations/de-DE.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    haveHeader: 'verwende Http Header',
+    clearField: 'lösche Feld',
+    dontSeeElementInDOM: 'sehe kein Element in DOM',
+    moveCursorTo: 'bewege den Cursor zu',
+    scrollTo: 'scrolle zu',
+    sendGetRequest: 'mache einen Get Request',
+    sendPutRequest: 'mache einen Put Request',
+    sendDeleteRequest: 'mache einen Delete Request',
+    sendPostRequest: 'mache einen Post Request',
+    switchTo: 'wechlse in iframe'
```
[the remark this answers is not in the document]
Comment (hubidu) [reply translations/de-DE.js]: will update

**Comment by DavertMik** [comment]
Ough, you can't use spaces in method names :)
Please pick a syntax which fits better to German language:

1. 'mache einen Post Request' => `macheEinenPostRequest`
2. 'mache einen Post Request' => `mache_einen_post_request`

Both are valid. But no spaces pls!

**Comment by PeterNgTr** [comment]
I would prefer the 1 style for js, 2 style is for ruby or python.

**Comment by hubidu** [comment]
I am doing style 1 since all the other translation files are also style 1

**Pull request closed without merging**